### PR TITLE
Add app skeleton

### PR DESCRIPTION
**Pull request by g-k**
Initial app skeleton and docker image

Runs locally with:

`poetry install`

Then run locally with: `poetry shell` then `SECRET_KEY=$(openssl rand -hex 32) ./manage.py runserver`
or build and run the docker images with: `SECRET_KEY=$(openssl rand -hex 32) docker-compose up --build`

### CODE REVIEW

**Comment by g-k** [comment]
r? @duallain 

do the install directions work for you?

File: README.md [hunk 1]
```diff
@@ -1,2 +1,25 @@
...
+### Installing
+
+1. Install the requirements:
+
+    1. [docker](https://docs.docker.com/get-docker/)
+    1. [docker-compose](https://docs.docker.com/compose/install/)
+    1. [Python 3.9](https://www.python.org/downloads/)
+    1. [poetry](https://python-poetry.org/docs/#installation)
+
+1. run `poetry install` to create a virtualenv and install python dependencies
```
Comment (duallain) [inline README.md:32]: For this to work I ended up needing to install postgres.  ```brew install postgresql``` was enough.  Error was basically that psycopg2 can't install without the pg_config binary.

**Review by duallain — APPROVED** [review]
Just one little comment. It looks good otherwise.  I really like starting with so many things setup well.  Docker build, poetry, postgres all setup, it's fantastic.

File: README.md (hunk 1 above)
Comment (g-k) [reply README.md:32]: Oh yeah, good catch! I must've already had those installed.

Updated the readme.

**Pull request merged**